app/controllers/new-signup.js

```diff
@@ -21,12 +21,18 @@ export default class NewSignUpController extends Controller {
   SECOND_STEP = NEW_SIGNUP_STEPS[1];
   THIRD_STEP = NEW_SIGNUP_STEPS[2];
   FOURTH_STEP = NEW_SIGNUP_STEPS[3];
-  LAST_STEP = NEW_SIGNUP_STEPS[4];
+  FIFTH_STEP = NEW_SIGNUP_STEPS[4];
+  LAST_STEP = NEW_SIGNUP_STEPS[5];
+
+  get isDevMode() {
+    return this.dev || false;
+  }
```

Comment:
This will be true for all values of dev, i.e if dev=false or xyz also this will be true?

Reply:
prakash I used service file instead doing...please check my latest commit